### ISSUE TITLE
Bugs in the UI for the Landing Page

### ISSUE THREAD

**Issue by abhay-dev2901** [anjalit03/Dazzle-and-Delight#20]
@anjalit03 I think I should work more on the UI enhancement of home page.
There are few issues which i have observed - 
1) Responsiveness in featured Events.

<img width="587" alt="Screenshot 2024-10-28 at 1 58 43 AM" src="https://github.com/user-attachments/assets/1016b823-3ba4-4b18-81d2-975fe8c46836">

2) The font of the homepage somewhere is not visible as it black and the background color is also with some darker theme, also the theme is not consistent throughout which contributes to bad user experience.
<img width="1439" alt="Screenshot 2024-10-28 at 1 58 24 AM" src="https://github.com/user-attachments/assets/afe4e3df-a225-4ebe-8666-0ebed30231e7">

3) The navbar is too enlage and inconsistent.
<img width="1440" alt="Screenshot 2024-10-28 at 1 56 21 AM" src="https://github.com/user-attachments/assets/174dec2d-1bac-472d-affd-cf5afc89225e">

Please assign me these tasks so that I can fix these issues under HacktoberFest 2024.

**Comment by anjalit03** [comment]
Yes please work on this but about the navbar someone is already working on this so please work on the other issues

**Comment by akash205sharma** [comment]
Plz assign to me

**Comment by abhay-dev2901** [comment]
@anjalit03 Please review PR #39 for the desired changes.
